### ISSUE TITLE
unable to send email, but able to receive email

### ISSUE THREAD

**Issue by psschand16** [Mailu/Mailu#2457]
## Similar issue 
(https://github.com/Mailu/Mailu/issues/1682)
https://github.com/Mailu/Mailu/issues/1200

### Environment
 - [ ] docker-compose

### Versions
version 1.9
```

## Description
I am able to receive email from my gmail, but unable to send any emails, I get delayed email notification. looking at the logs I can see a connection failed error in smtp container

mailu-smtp-1  | aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host 192.168.203.10:80 ssl:default [Connect call failed ('192.168.203.10', 80)]


mailu-antispam-1   | 2022-09-27 01:09:06 #7(rspamd_proxy) <bcbece>; proxy; proxy_milter_finish_handler: finished milter connection
mailu-front-1      | 2022/09/27 01:09:06 [info] 11#11: *322 client logged in, client: 192.168.203.11, server: 0.0.0.0:10025, login: "admin@kvjg.tk", upstream: 192.168.203.7:10025
mailu-smtp-1       | 2022-09-27T01:09:06.482131+00:00 31badb458a96 postfix/smtpd[376]: connect from mailu-front-1.mailu_default[192.168.203.2]
mailu-smtp-1       | 2022-09-27T01:09:06.482860+00:00 31badb458a96 postfix/smtpd[376]: discarding EHLO keywords: PIPELINING
mailu-smtp-1       | 2022-09-27T01:09:06.483656+00:00 31badb458a96 postfix/smtpd[376]: discarding EHLO keywords: PIPELINING
mailu-smtp-1       | 2022-09-27T01:09:06.568926+00:00 31badb458a96 postfix/smtpd[376]: 8ADF1473A35: client=mailu-webmail-1.mailu_default[192.168.203.11], sasl_method=XCLIENT, sasl_username=admin@kvjg.tk
mailu-smtp-1       | 2022-09-27T01:09:06.601504+00:00 31badb458a96 postfix/cleanup[377]: 8ADF1473A35: message-id=<2e136627c81172b3bb3bb4b30c049941@kvjg.tk>
mailu-smtp-1       | 2022-09-27T01:09:06.835170+00:00 31badb458a96 postfix/qmgr[360]: 8ADF1473A35: from=<admin@kvjg.tk>, size=572, nrcpt=1 (queue active)
mailu-antispam-1   | 2022-09-27 01:09:06 #7(rspamd_proxy) <76259b>; proxy; proxy_milter_finish_handler: finished milter connection
mailu-smtp-1       | 2022-09-27T01:09:06.837428+00:00 31badb458a96 postfix/smtpd[376]: disconnect from mailu-webmail-1.mailu_default[192.168.203.11] ehlo=2 xclient=0/1 mail=1 rcpt=1 data=1 quit=1 commands=6/7
mailu-webmail-1    | sendmail: <q8kmdr11> User admin@kvjg.tk [192.168.203.2]; Message <2e136627c81172b3bb3bb4b30c049941@kvjg.tk> for andy16@gmail.com; 250: 2.0.0 Ok: queued as 8ADF1473A35
mailu-smtp-1       | 2022-09-27T01:09:06.835170+00:00 31badb458a96 postfix/qmgr[360]: 8ADF1473A35: from=<admin@kvjg.tk>, size=572, nrcpt=1 (queue active)
mailu-antispam-1   | 2022-09-27 01:09:06 #7(rspamd_proxy) <76259b>; proxy; proxy_milter_finish_handler: finished milter connection
mailu-smtp-1       | 2022-09-27T01:09:06.837428+00:00 31badb458a96 postfix/smtpd[376]: disconnect from mailu-webmail-1.mailu_default[192.168.203.11] ehlo=2 xclient=0/1 mail=1 rcpt=1 data=1 quit=1 commands=6/7
mailu-smtp-1       | 2022-09-27T01:09:37.064008+00:00 31badb458a96 postfix/smtp[362]: connect to gmail-smtp-in.l.google.com[108.177.15.26]:25: Operation timed out
mailu-antispam-1   | 2022-09-27 01:09:39 #7(rspamd_proxy) <fc8835>; proxy; proxy_accept_socket: accepted milter connection from 192.168.203.7 port 45334
mailu-antispam-1   | 2022-09-27 01:09:39 #7(rspamd_proxy) <fc8835>; milter; rspamd_milter_process_command: got connection from 127.0.0.1:33719
mailu-antispam-1   | 2022-09-27 01:09:39 #7(rspamd_proxy) <fc8835>; proxy; proxy_milter_finish_handler: finished milter connection
mailu-front-1      | 198.235.24.143 - - [27/Sep/2022:01:09:55 +0000] "GET / HTTP/1.1" 301 162 "-" "Expanse, a Palo Alto Networks company, searches across the global IPv4 space multiple times per day to identify customers&#39; presences on the Internet. If you would like to be excluded from our scans, please send IP addresses/domains to: scaninfo@paloaltonetworks.com"
mailu-front-1      | 198.235.24.143 - - [27/Sep/2022:01:09:55 +0000] "GET / HTTP/1.1" 301 162 "http://132.145.61.53:80/" "Expanse, a Palo Alto Networks company, searches across the global IPv4 space multiple times per day to identify customers&#39; presences on the Internet. If you would like to be excluded from our scans, please send IP addresses/domains to: scaninfo@paloaltonetworks.com"
mailu-front-1      | 198.235.24.143 - - [27/Sep/2022:01:09:55 +0000] "GET /webmail/ HTTP/1.1" 302 138 "-" "Expanse, a Palo Alto Networks company, searches across the global IPv4 space multiple times per day to identify customers&#39; presences on the Internet. If you would like to be excluded from our scans, please send IP addresses/domains to: scaninfo@paloaltonetworks.com"
mailu-front-1      | 198.235.24.143 - - [27/Sep/2022:01:09:55 +0000] "GET /sso/login HTTP/1.1" 200 6596 "-" "Expanse, a Palo Alto Networks company, searches across the global IPv4 space multiple times per day to identify customers&#39; presences on the Internet. If you would like to be excluded from our scans, please send IP addresses/domains to: scaninfo@paloaltonetworks.com"
mailu-imap-1       | Sep 27 01:10:02 pop3-login: Info: Disconnected: Aborted login by logging out (no auth attempts in 0 secs): user=<>, rip=127.0.0.1, lip=127.0.0.1, secured, session=<g+PCTZ7p0Zp/AAAB>
mailu-smtp-1       | 2022-09-27T01:10:07.098773+00:00 31badb458a96 postfix/smtp[362]: connect to alt1.gmail-smtp-in.l.google.com[142.250.153.27]:25: Operation timed out
mailu-smtp-1       | 2022-09-27T01:10:37.134751+00:00 31badb458a96 postfix/smtp[362]: connect to alt2.gmail-smtp-in.l.google.com[142.251.9.27]:25: Operation timed out
mailu-antispam-1   | 2022-09-27 01:10:39 #7(rspamd_proxy) <1871cb>; proxy; proxy_accept_socket: accepted milter connection from 192.168.203.7 port 51382
mailu-antispam-1   | 2022-09-27 01:10:39 #7(rspamd_proxy) <1871cb>; milter; rspamd_milter_process_command: got connection from 127.0.0.1:43887
mailu-antispam-1   | 2022-09-27 01:10:39 #7(rspamd_proxy) <1871cb>; proxy; proxy_milter_finish_handler: finished milter connection
mailu-imap-1       | Sep 27 01:11:02 pop3-login: Info: Disconnected: Aborted login by logging out (no auth attempts in 0 secs): user=<>, rip=127.0.0.1, lip=127.0.0.1, secured, session=<dS1ZUZ7pm4l/AAAB>
mailu-smtp-1       | 2022-09-27T01:11:07.146869+00:00 31badb458a96 postfix/smtp[362]: connect to alt3.gmail-smtp-in.l.google.com[142.250.150.27]:25: Operation timed out
mailu-smtp-1       | 2022-09-27T01:11:37.159897+00:00 31badb458a96 postfix/smtp[362]: connect to alt4.gmail-smtp-in.l.google.com[74.125.200.26]:25: Operation timed out
mailu-smtp-1       | 2022-09-27T01:11:37.166446+00:00 31badb458a96 postfix/smtp[362]: 8ADF1473A35: to=<andy16@gmail.com>, relay=none, delay=151, delays=0.33/0.03/150/0, dsn=4.4.1, status=deferred (connect to alt4.gmail-smtp-in.l.google.com[74.125.200.26]:25: Operation timed out)
mailu-smtp-1  | 2022-09-24T19:41:48.304335+00:00 d8b184265a96 postfix/lmtp[5025]: 3053C4790AC: to=<admin@kvjg.tk>, relay=192.168.203.10[192.168.203.10]:2525, delay=0.11, delays=0.02/0.03/0.01/0.05, dsn=2.0.0, status=sent (250 2.0.0 <admin@kvjg.tk> oyZjD3xdL2PMFAAAWXdD1g Saved)
mailu-smtp-1  | 2022-09-24T19:41:48.304683+00:00 d8b184265a96 postfix/qmgr[340]: 3053C4790AC: removed
mailu-smtp-1  | 2022-09-24T19:41:48.306719+00:00 d8b184265a96 postfix/smtp[5011]: connect to gmail-smtp-in.l.google.com[142.250.110.26]:25: Operation timed out
mailu-smtp-1  | 2022-09-24T19:41:48.306723+00:00 d8b184265a96 postfix/smtp[5009]: connect to gmail-smtp-in.l.google.com[142.250.110.26]:25: Operation timed out
mailu-smtp-1  | 2022-09-24T19:41:48.306764+00:00 d8b184265a96 postfix/smtp[5009]: connect to gmail-smtp-in.l.google.com[2a00:1450:400c:c0d::1a]:25: Address not available
mailu-smtp-1  | 2022-09-24T19:41:48.306766+00:00 d8b184265a96 postfix/smtp[5011]: connect to gmail-smtp-in.l.google.com[2a00:1450:400c:c0d::1a]:25: Address not available
mailu-smtp-1  | 2022-09-24T19:41:48.315488+00:00 d8b184265a96 postfix/smtp[5011]: connect to alt1.gmail-smtp-in.l.google.com[2a00:1450:4013:c16::1a]:25: Address not available
mailu-smtp-1  | 2022-09-24T19:41:48.317550+00:00 d8b184265a96 postfix/smtp[5009]: connect to alt1.gmail-smtp-in.l.google.com[2a00:1450:4013:c16::1a]:25: Address not available


## Replication Steps
Running Mailu docker compose file on a fresh ubuntu  ARM 64 VPS instance

## Expected behavior
Mail should be sent

## Logs

````markdown
```
I have attached smtp logs from the command
docker-compose logs -f smtp
```
````
[smtp.log](https://github.com/Mailu/Mailu/files/9640491/smtp.log)

**Comment by nextgens** [comment]
The important line is:
```postfix/smtp[362]: connect to gmail-smtp-in.l.google.com[108.177.15.26]:25: Operation timed out```

It means that something is preventing your Mailu instance from reaching google's email server. It may be your VPS provider that filters access to TCP port 25 outbound.

**Comment by andriusign** [comment]
I am encountering the same issue. What is the problem causing this and how can it be fixed?

**Comment by nextgens** [comment]
Check with your ISP

**Comment by andriusign** [comment]
Check what? ISP does not block anything for me.

**Comment by nextgens** [comment]
Are you sure that your ISP doesn't block anything for you? Try connecting with netcat and see if that works any better.

```$nc -C gmail-smtp-in.l.google.com 25
220 mx.google.com ESMTP bq27-20020a5d5a1b000000b0022e3271dd72si6144184wrb.356 - gsmtp
QUIT
221 2.0.0 closing connection bq27-20020a5d5a1b000000b0022e3271dd72si6144184wrb.356 - gsmtp
```

If you don't get the banner back from google something is blocking your request; odds are it's your ISP if it's not you.


Server to server communication has always been on port 25; servers can (and do) negotiate using STARTTLS when they want security.

**Comment by andriusign** [comment]
Actually not, the problem was not 25 port, because after ISP unblocked it, Mailu still does not send email and fail with errors. This does not happen with Mailcow, so I assume Mailu's settings are incorrect.

**Comment by yusoffnike01** [comment]
I get the same issue,  how to solve this ?

**Comment by nextgens** [comment]
Check that you can access other SMTP servers, as described above with gmail. If you can't "something" is blocking it, odds are it's your ISP.
